examples/new-cli/README.md

```diff
@@ -0,0 +1,89 @@
+```
+This will override the options in the yaml (be sure to change the `trajectory_directory` so you don't overwrite your previous simulation.
+Currently only `key:value` parts of the yaml can be overridden i.e. not sequences or lists.
+
+To view all options ultimately used in the simulation, a file named `parsed-$date-.yaml` is created. 
+This file is located in the same directory as the yaml file used to launch the simulation.
+
+## Example Use Case
+
+In this example folder we have a protein: [2ZFF](https://www.rcsb.org/structure/2zff) and some ligands which we will use for a series of free energy calculations.
+We will use a geometric based atom mapping.
+Our yaml file `my.yaml` is setup to do an alchemical transformation from ligand 0 to ligand 1 for the solvent, vacuum, and complex phase.
+We will use a bash loop (see `run_star_map.sh`) to do a star map with 6 different ligands.
+We will put the ligand at index 0 at the center of star map with the following bash script:
+```bash
+#!/usr/bin/env bash
+
+for new_ligand_idx in $(seq 1 5)
+do 
+	perses-cli --yaml my.yaml --override old_ligand_index:0 --override new_ligand_index:"$new_ligand_idx" --override trajectory_directory:lig0to"$new_ligand_idx"
+done
+```
+This is equivalent to running all of these commands manually:
+
+```bash
+$ perses-cli --yaml my.yaml --override old_ligand_index:0 --override new_ligand_index:1 --override trajectory_directory:lig0to1
+$ perses-cli --yaml my.yaml --override old_ligand_index:0 --override new_ligand_index:2 --override trajectory_directory:lig0to2
+$ perses-cli --yaml my.yaml --override old_ligand_index:0 --override new_ligand_index:3 --override trajectory_directory:lig0to3
+$ perses-cli --yaml my.yaml --override old_ligand_index:0 --override new_ligand_index:4 --override trajectory_directory:lig0to4
+$ perses-cli --yaml my.yaml --override old_ligand_index:0 --override new_ligand_index:5 --override trajectory_directory:lig0to5
+```
+
+## Analysis 
+TODO
+- notebook or script
+- sdf file has tags that contain exp data
+
+## Docker Example
+
+First, grab the dev image of perses that has the new CLI tool with `docker pull choderalab/perses:dev`.
+General docker instructions for using perses and docker can be found [here](https://github.com/choderalab/perses/tree/main/docker#readme).
+To run this example use the following docker command (to make it easier to read the command is split across multiple lines but this is not necessary).
+```bash
+docker run -it --rm --gpus device=0 --mount type=bind,source=$HOME/.OpenEye/,target=/openeye/,readonly \
+                                    --mount type=bind,source=$HOME/repos/perses/examples/,target=/mnt/ \
+                                    -w /mnt/new-cli choderalab/perses:dev bash ./run_star_map.sh
+```
+
+Of importance there are the paths to the OpenEye license file (in this example is `$HOME/.openeye/`), path to the examples directory in perses (`$HOME/Projects/perses/examples/`)
```

Comment:
The docker command runs locally! Great work!

Reply:
I agree we should be consistent, this page here: https://github.com/choderalab/perses/tree/main/docker has a mix 
`type=bind,source=$HOME/.openeye/,target=/openeye/` and `type=bind,source=$HOME/.OpenEye/,target=/openeye/,readonly` I think since the opeye docs https://docs.eyesopen.com/toolkits/python/quickstart-python/license.html say `~USERNAME/.OpenEye` then we should change everything to match that. I will do that in this PR.